core/src/states/mod.rs

```diff
@@ -505,9 +517,9 @@ impl<T: Owner + std::fmt::Debug + 'static> StateManager<T> {
                 changed_kickoff_machines.extend(std::mem::take(&mut ctx.new_kickoff_machines));
             }
 
-            if iterations > 500 {
+            if iterations > 3000 {
```

Comment:
set to 10000 and add // TODO: https://github.com/chainwayxyz/clementine/issues/675